packages/base/src/UI5Element.js

```diff
@@ -267,6 +268,10 @@ class UI5Element extends HTMLElement {
 				this._attachChildPropertyUpdated(child, slotData.listenFor);
 			}
 
+			if (child.isUI5Element && slotData.invalidateParent) {
+				child.shouldInvalidateParent = slotData.invalidateParent;
```

Comment:
`child._shouldInvalidateParent = true;` is a bit more straightforward, becuase if it is falsy, we'll never get here anyway